js/views/modals/editListing/EditListing.js

```diff
@@ -893,14 +893,13 @@ export default class extends BaseModal {
     const excludes = '.js-sectionShipping, .js-couponsSection, .js-variantsSection, ' +
       '.js-variantInventorySection';
 
-    return this._$formFields ||
```

Comment:
The issue was that the quantity field is being dynamically rendered in after making a changing in its adjacent drop down and the form fields selector is already cached before that. Just going to turn off the cache for now.